🗒️ update react-server's doc to mentioned how to customized error page (#1847)
Shopify/quilt: packages/react-server/docs/webpack-plugin.md

}
```

### Error component

> There is a default error page used to display production SSR errors build into this library.

Reviewer: ```suggestion
There is a default error page used to display production SSR errors built into this library.
```

> This display is customizable with `renderError` option while using react-server's `createServer` or `createRender` funciton.

Reviewer: ```suggestion
This can be customized using the `renderError` option for `createServer` or `createRender`.
```

To customized this error display when using React Server Webpack Plugin, add a error.ts/tsx/jsx file with default export in your project's react-server `basePath` (typically where your index.ts/tsx/jsx is).

> A virtual entry point file `error.entry.client.js` will be added for you when a error.ts/tsx/jsx file exist, which sewing-kit will pick up as an entry point for SSR error display.

Reviewer: ```suggestion
A virtual entry point file `error.entry.client.js` will be built at compile time for you when a `error.ts`/`tsx`/`jsx` file exist, which the webpack plugin will pick up as an entry point for SSR error display.
```
